drivers/include/net/netdev.h

```diff
@@ -54,6 +54,7 @@ extern "C" {
 #include <sys/uio.h>
 
 #include "net/netstats.h"
+#include "net/l2filter.h"
```

Comment:
hm, thinking about @smlng's comment: including header files, where the implementation is known to be not present, may still lead to name clashes if those functions / variables are too generic in their names and in the worst case to undefined behavior (I assume there is no big #ifdef in the header file itself). IMO it's also better to take off some work for the preprocessor / compiler by hiding those includes, iff its known that they are not needed. But as @miri64 states, it could get ugly and bloated very quickly.